library/Icingadb/Widget/Detail/EventDetail.php

```diff
@@ -199,39 +206,50 @@ protected function assembleStateChangeEvent(StateHistory $stateChange)
             ]);
         }
 
-        $this->addHtml(
+        $eventInfo = [
             new HtmlElement('h2', null, Text::create(t('Event Info'))),
             new HorizontalKeyValue(t('Occurred On'), DateFormatter::formatDateTime($stateChange->event_time)),
             new HorizontalKeyValue(t('Scheduling Source'), $stateChange->scheduling_source),
             new HorizontalKeyValue(t('Check Source'), $stateChange->check_source)
-        );
+        ];
+
         if ($stateChange->state_type === 'soft') {
-            $this->addHtml(new HorizontalKeyValue(t('Check Attempt'), sprintf(
+            $eventInfo[] = new HorizontalKeyValue(t('Check Attempt'), sprintf(
                 t('%d of %d'),
                 $stateChange->attempt,
                 $stateChange->max_check_attempts
-            )));
+            ));
         }
-        $this->addHtml(
-            new HorizontalKeyValue(t('State'), $stateChange->object_type === 'host'
+
+        $eventInfo[] = new HorizontalKeyValue(
+            t('State'),
+            $stateChange->object_type === 'host'
                 ? ucfirst(HostStates::text($objectState))
-                : ucfirst(ServiceStates::text($objectState))),
-            new HorizontalKeyValue(
-                t('State Type'),
-                $stateChange->state_type === 'hard' ? t('Hard', 'state') : t('Soft', 'state')
-            ),
-            new HorizontalKeyValue($objectKey, $objectInfo)
+                : ucfirst(ServiceStates::text($objectState))
+        );
+
+        $eventInfo[] = new HorizontalKeyValue(
+            t('State Type'),
+            $stateChange->state_type === 'hard' ? t('Hard', 'state') : t('Soft', 'state')
         );
+
+        $eventInfo[] = new HorizontalKeyValue($objectKey, $objectInfo);
+
+
```

Comment:
Why two empty lines? To set it off of the single ones above? Why they're there? Empty lines are only mandatory after control structures, *most* others are personal style (I doubt that it is that, because in the method below there are none), but two or more subsequent empty lines are discouraged.

Reply:
That was a mistake, I just wanted to add a new line.